src/accelerator.rs

```diff
@@ -0,0 +1,87 @@
+use drivers::Driver;
+use motions::Motion;
+
+fn clamp(n: isize, min: isize, max: isize) -> isize {
+  if n < min {
+    return min;
+  }
+  if n > max {
+    return max;
+  }
+  n
```

Comment:
yup... not all rust statements need one (this mostly applies to one liners though)